src/sync.c

```diff
@@ -202,9 +210,12 @@ void sync_process(sync_t *st, float complex *buffer)
 
     for (i = 0; i < partitions_per_band * 19 + 1; i += 19)
     {
+        prev_slope[LB_START + i] += st->angle_adj;
```

Comment:
These two lines have an unintended side-effect during startup due to line 53. Line 53 is a hack that uses prev_slope[...] = 0 has a flag that we don't have any previous slope information.

Reply:
I'll check into that tonight, but I suspect there will be phase information by the time this line is reached. It could be there's a better way to implement it though.

Reply:
If you look at my gist I linked, my solution was to fix the hack at line 53. These lines should be fine, but their addition made the hack problematic.

Reply:
> by the time this line is reached

I meant to say: by the time `st->angle_adj` is non-zero.